README.md

```diff
@@ -31,140 +31,192 @@ This repository hosts an e-commerce project tailored for baby products. It is de
 
 2. **Additional Configurations:**
 
-   - **Customizing Ports:** Modify `EXPOSE` and the `runserver` port in `ENTRYPOINT` to use a different port if needed.
-   - **Environment Variables:** Use a `COPY` command to include the `.env` file or configure them in your deployment environment.
-   - **Database Configuration:** Update the `DATABASES` setting in `settings.py` to use a production database like PostgreSQL or MySQL.
+   - **Environment Variables:** Pass environment variables dynamically using the `--env-file` option in the `docker run` command instead of relying on volumes for the .env file. This allows flexibility for different containers.
 
 3. **Build the Docker image:**
-   ```bash
-   docker build -t {project_name} -f Dockerfile .
-   ```
-4. **Run the container:**
 
-   ```bash
-   docker run -it --rm -p 8025:8025 {project_name}
-   ```
+```bash
+# Bash
+docker build -t {project_name} -f Dockerfile .
+```
 
-   **Hint:** Use the `-v` flag with `docker run` to persist data and avoid data loss across restarts. For example:
+4. **Run the container with a specific .env file:**
 
-   ```bash
-   docker run -d --restart on-failure -p 8025:8025 \
-   -v /baby-tools-shop/project_images:/app/project_images \
-   -v /baby-tools-shop/.env:/app/.env \
-   baby-tools-shop
-   ```
+```bash
+# Bash
+docker run -it --rm -p 8025:8025
+-v /baby-tools-shop/project_images:/app/project_images
+--env-file {path_to_env_file} {project_name}
+```
 
-   - The `-v` flag ensures that important directories like `project_images` are linked to persistent storage on the host.
-   - The `--restart on-failure` option ensures that the container restarts automatically if it crashes.
-   - The `-d` flag stands for detached mode, not showing terminal statements
+**Hint:** Use `--env-file` to specify different environment variable files for different containers. This avoids hardcoding environment-specific configurations.
 
 ### Deployment on Virtual Machine
 
 1. **Clone the repository on VM:**
-   ```bash
-   git clone https://github.com/your-repo-name
-   ```
-2. **Copy `.env` file to the VM:**
+
+```bash
+# Bash
+git clone https://github.com/your-repo-name
+```
+
+2. **Copy .env file to the VM:**
    Ensure it contains the necessary variables.
+
 3. **Build Docker image on the VM:**
-   ```bash
-   docker build -t {project_name} -f Dockerfile .
-   ```
-4. **Run the container:**
-   ```bash
-   docker run -d --restart on-failure -p 8025:8025 \
-   -v /baby-tools-shop/project_images:/app/project_images \
-   -v /baby-tools-shop/.env:/app/.env \
-   baby-tools-shop
-   ```
-5. **Access Apllication**
-   Access the application: Open your browser and navigate to http://{vmip}:8025/
+
+```bash
+# Bash
+docker build -t {project_name} -f Dockerfile .
+```
+
+4. **Run the container with a specific .env file:**
+
+```bash
+# Bash
+docker run -d --restart on-failure -p 8025:8025
```

Comment:
The `\`are very important in Long formatted commando lines, because allows a new Linebreak.

The right version should be this one: 
```bash
   docker run -d --restart on-failure -p 8025:8025 \
   -v /baby-tools-shop/project_images:/app/project_images \
   -v /baby-tools-shop/.env:/app/.env \
   baby-tools-shop
   ```